src/service/resources-impl.js

```diff
@@ -1628,16 +1655,15 @@ export class Resource {
   /**
    * Requests the resource's element to be built. See {@link AmpElement.build}
    * for details.
-   * @param {boolean} force
    * @return {boolean}
```

Comment:
Nevermind, I got what you were saying. Just return instead of return false/true.